src/adr/0026-removing-configured-additional-mailboxListeners.md

```diff
@@ -19,17 +19,33 @@ binds to the main event exchange.
 If the user unconfigures the listener, the queue and the binding are still present but not consumed. This results in 
 unbounded queue growth eventually causing RabbitMQ resource exhaustion and failure.
 
+## Vocabulary
+
+A **required group** is a group configured within James additional mailbox listener or statically binded via Guice. We 
+should have a queue for that mailbox listener binded to the main exchange.
+
+A **registered group** is a group whose queue exists in RabbitMQ and is bound to the exchange, independently of its James 
+usage. If it is required, a consumer will consume the queue. Otherwise the queue might grow unbounded.
+
 ## Decision
 
-For rabbitMQ, configuration changes of additional mailbox listeners should be tracked via event sourcing. Event sourcing is 
+For RabbitMQ, configuration changes of additional mailbox listeners should be tracked via event sourcing. Event sourcing is 
 desirable as it allows:
  - Detecting previously removed MailboxListener upon start
  - Audit of unbind decisions
  - Enables writing more complex business rules in the future
 
-We need, upon start, to sanitize bindings, and remove the ones corresponding to mailboxListeners that were removed not configured.
+We need a clear consensus and auditability across the James cluster about (and their changes). Thus Event sourcing will 
+maintain an aggregate tracking **required groups** (and their changes). Audit will be enabled by adding host and date 
+information upon changes. A subscriber will perform changes (binds and unbinds) in registered groups following the 
```

Comment:
It's not clear to me. For me the bind and unbind of a registered group should be actions done before the emission of the Events. Otherwise we are doing command sourcing and not event sourcing.

Reply:
Description of commands & events done.